test/unit/lib/vehicle.js

```diff
@@ -489,53 +472,112 @@ test('charge - no age', async function(t) {
   const response = await vehicle.charge();
   t.deepEqual(response.data, body);
   t.is(response.age, null);
-
 });
 
 test('vin', async function(t) {
-
   const body = {
     vin: '4JGBB8GB2AA537355',
   };
-  t.context.n = nocks.base()
+  t.context.n = nocks
+    .base()
     .get('/vin')
     .reply(200, body);
 
   const vin = await vehicle.vin();
   t.is(vin, body.vin);
-
 });
 
 test('lock', async function(t) {
-
-  t.context.nock = nocks.base()
+  t.context.nock = nocks
+    .base()
     .post('/security', {action: 'LOCK'})
     .reply(200, {status: 'success'});
 
   const response = await vehicle.lock();
 
-  t.deepEqual(
-    _.xor(_.keys(response), [
-      'status',
-    ]),
-    []
-  );
+  t.deepEqual(_.xor(_.keys(response), ['status']), []);
   t.is(response.status, 'success');
 });
 
 test('unlock', async function(t) {
-
-  t.context.nock = nocks.base()
+  t.context.nock = nocks
+    .base()
     .post('/security', {action: 'UNLOCK'})
     .reply(200, {status: 'success'});
 
   const response = await vehicle.unlock();
 
-  t.deepEqual(
-    _.xor(_.keys(response), [
-      'status',
-    ]),
-    []
-  );
+  t.deepEqual(_.xor(_.keys(response), ['status']), []);
   t.is(response.status, 'success');
 });
+
+test('batch', async function(t) {
+  vehicle.setUnitSystem('imperial');
+  const paths = ['/odometer', '/transmission/fluid', '/fuel', '/sunroof'];
+  const requestBody = {
+    headers: {
+      'sc-unit-system': 'imperial',
+    },
+    requests: [
+      {
+        path: '/odometer',
+      },
+      {
+        path: '/transmission/fluid',
+      },
+      {
+        path: '/fuel',
+      },
+      {
+        path: '/sunroof',
+      },
+    ],
+  };
+  const responseBody = {
```

Comment:
```suggestion
  const expectedResponseBody = {
```

Reply:
Rename it so its more clear that this is the expected response body and not the actual response body.